Makefile

```diff
@@ -40,7 +41,11 @@ uitest: ## 🧪 Run the ui tests in headless mode
 
 build-frontend: ## 🏗️ Build the Frontend webapp
 	@echo -e "\e[34m$@\e[0m" || true
-	@cd code/frontend && npm install && npm run build
+	@cd code/frontend && npm install && VITE_ENV_DIR=$(AZURE_ENV_DIR) npm run build
+
+unittest-frontend: build-frontend ## 🏗️ Unit test the Frontend webapp
```

Comment:
We don't need 2 `unittest-frontend` targets

Reply:
That was by mistake. I will remove the old one